modules/lvm2/udiskslvm2moduleiface.c

```diff
@@ -154,7 +154,6 @@ lvm_update_from_variant (GPid      pid,
           udisks_linux_volume_group_object_destroy (group);
           g_dbus_object_manager_server_unexport (manager,
                                                  g_dbus_object_get_object_path (G_DBUS_OBJECT (group)));
-          g_object_unref (G_OBJECT (group));
```

Comment:
I wonder if this code shouldn't ``ref()`` the object instead when it gets the reference above.